### PR TITLE
Updated bash script and documentation for mutation-based fuzzing option

### PR DESCRIPTION
**Pull request by mingxguo27**

### CODE REVIEW

File: docs/fuzzing.md [hunk 1]
```diff
@@ -13,12 +13,14 @@ vulnerabilities are not observable, and even the crash behaviour varies from cas
...
-## Corpus testing
+## How does it work
 
 Our idea is fuzzing by proxy. We take [OpenSK](https://github.com/google/OpenSK)
 (an open source implementation of a FIDO2 security key) as our fuzz target and 
 generate interesting input data guided by OpenSK's code coverage. At the moment,
-the fuzzing tool consists of running this input corpus on the device under test.
+we support two fuzzing modes: 
+1. Run this input corpus directly on the device under test.
+2. Fuzz the device under test using this input corpus as initial corpus.
```
Comment (jennypullman) [inline docs/fuzzing.md:23]: It would be the initial corpus, but then it would also be used for creating new data input right? (using the mutator?)

Comment (jennypullman) [reply docs/fuzzing.md:23]: Maybe you could say " Fuzz the device under test using mutations of this input corpus as initial corpus."

**Review by jennypullman — APPROVED** [review]

File: docs/fuzzing.md [hunk 2]
```diff
@@ -55,6 +59,18 @@ For more control, the following arguments are available:
 - `--port`: If a GDB monitor is selected, the port to listen on for GDB remote 
   connection.
 
+In the fuzzing mode, more options are supported:
```
Comment (kaczmarczyck) [inline docs/fuzzing.md:62]: I think terminology is confusing here. Isn't everything fuzzing?

File: docs/fuzzing.md (hunk 1 above)
Comment (kaczmarczyck) [inline docs/fuzzing.md:23]: I think the striking difference should be named here: One is stopping with a result file, the other is not stopping. Both are called fuzzing, or the instructions don't match with user expectations on a file called `fuzzing.md`.

Comment (kaczmarczyck) [reply docs/fuzzing.md]: I changed the wording to something similar.

Comment (kaczmarczyck) [reply docs/fuzzing.md]: I missed the comment above, resolved together.

File: docs/fuzzing.md (hunk 2 above)
Comment (kaczmarczyck) [reply docs/fuzzing.md]: Changed the wording.

**Review by jennypullman — APPROVED** [review]

**Pull request merged**